_data/simple-icons.json

```
@@ -5666,6 +5666,14 @@
 		"hex": "4285F4",
 		"source": "https://flood.io"
 	},
+	{
+		"title": "Floorp",
+		"hex": "3D0AE8",
```

```suggestion
		"hex": "5309E8",
```

Feedback from @CutterKnife